schema/revert/shipping/views.sql

```diff
@@ -1158,6 +1159,7 @@ create or replace view shipping.scan_encounters_v1 as
         age_in_years(upper(age_bin_coarse_v2.range)) as age_range_coarse_upper,
 
         location.hierarchy -> 'puma' as puma,
+        location.hierarchy -> 'tract' as census_tract,
```

Comment:
did this slip in here x2 ?

Reply:
Ditto auto-generated revert script.

Reply:
Yep, makes sense to me because it's `revert` not `deploy`.